meetings/2020-02-05.md

```diff
@@ -0,0 +1,81 @@
+# Node.js Technical Steering Committee (TSC) Meeting 2020-02-05
+
+## Links
+
+* **Recording**: https://www.youtube.com/watch?v=p5fIPf94GuY
+* **GitHub Issue**: https://github.com/nodejs/TSC/issues/810
+* **Minutes Google Doc**: https://docs.google.com/document/d/1OmKotKXE12zGi3sTa8EkGdM4Ocpkrgy2M8BuFhTIK_0
+
+## Present
+
+* Anna Henningsen @addaleax (TSC)
+* Beth Griggs @BethGriggs (TSC)
+* Сковорода Никита Андреевич @ChALkeR (TSC)
+* Daniel Bevenius @danbev (TSC)
+* James Snell @jasnell (TSC)
+* Matteo Collina @mcollina (TSC)
+* Michaël Zasso @targos (TSC)
+* Rich Trott @Trott (TSC)
+* Sam Roberts @sam-github (TSC)
+* Tobias Nießen @tniessen (TSC)
+* Vladimir de Turckheim (Guest)
+
+## Agenda
+
+### Announcements
+
+* Nominating Ruben Bridgewater (@BridgeAR) to the TSC [#811](https://github.com/nodejs/TSC/issues/811)
+  * Will be invited as an observer to the next meeting.
```

Comment:
Opened https://github.com/nodejs/create-node-meeting-artifacts/pull/83 to add him to the invited list. Not sure if anyone other than @mhdawson can merge it.